src/components/BookItem.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+
+const BookItem = ({ item }) => (
```

Comment:
Great job @mwismann 🤝

- [ ] Kindly ensure each book item has a button to remove the book from the list of books.